### PR TITLE
Release 4.1.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 4.1.1 (2020-07-07)
+- Fixed issue where fuzzy date could return incorrect data when sending in a day value of the 31st. [#181](https://github.com/blackbaud/skyux-datetime/pull/181) (Thanks [@Blackbaud-CoreyArcher](https://github.com/Blackbaud-CoreyArcher)!)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:2]: Would this work?

`Fixed an issue where the fuzzy date pipe returns incorrect data when the day value is 31.`

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.1](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.1)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md:2]: I like the latter.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md:2]: Normally I would go with the second one as it is more concise. However, in this case the 31 date didn't seem to _always_ break and so lets go with the first :) I'll update.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: @blackbaud-johnly updated :)

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I didn't see Alex's message until after my comments were added. I'm fine if @blackbaud-johnly feels like we should use the first. Feel free to update it to that if you like that better.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: I liked the second one better, but either one worked for me, so I just approved the PR. Thanks!

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.1](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-release-4.1.1)

**Pull request merged**